src/components/shared/card/index.tsx

```diff
@@ -67,7 +71,12 @@ const Card = ({
     >
       <div className='flex min-w-0 gap-4'>
         <div className='flex flex-col justify-between'>
-          <CardThumbnail thumbnail={thumbnail} title={title} />
+          <CardThumbnail
+            isClosed={isClosed}
+            isPending={isPending}
+            thumbnail={thumbnail}
+            title={title}
+          />
 
           <div className='flex flex-1 items-center'>
```

Comment:
```suggestion
          <div className='flex items-center'>
```
이건 그냥 궁굼한건데 왜 여기에 flex-grow 준거에용?
위에 왼쪽 < 구간을 담당하는 div 에서 grow-1 줘야하는거 아닌가 하는데
틀렸다면 저를 욕해주세요!